binrw_derive/src/parser/types/passed_args.rs

```diff
@@ -7,7 +7,7 @@ pub(crate) enum PassedArgs {
     None,
     List(Vec<TokenStream>),
     Tuple(TokenStream),
-    Named(Vec<(syn::Ident, Option<syn::Expr>)>),
+    Named(Vec<TokenStream>),
```

Comment:
This feels weird, though it doesn't affect much. may be worth changing such that it's just TokenStream instead of Vec<TokenStream>